external/akka/src/main/scala/org/apache/spark/streaming/akka/ActorReceiver.scala

```diff
@@ -105,13 +108,25 @@ abstract class ActorReceiver extends Actor {
   }
 
   /**
-   * Store a single item of received data to Spark's memory.
+   * Store a single item of received data to Spark's memory asynchronously.
    * These single items will be aggregated together into data blocks before
    * being pushed into Spark's memory.
    */
   def store[T](item: T) {
     context.parent ! SingleItemData(item)
   }
+
+  /**
+   * Store a single item of received data to Spark's memory synchronously.
+   * These single items will be aggregated together into data blocks before
+   * being pushed into Spark's memory.
+   *
+   * As opposed to [[ActorReceiver.store[T]: Unit]], this method allows flow control
+   * (maxRate, backpressure) to block the input.
+   */
+  def store[T](item: T, timeout: Timeout): Future[Any] = {
```

Comment:
Just use `f.map(_ => {})(context.dispatcher)`? Comparing what we did in `store(msg.asInstanceOf[T])`, I think the `map` cost doesn't matter.